Data/DataModel/Response.cs

```diff
@@ -69,6 +69,21 @@ public string SubmittedText
         /// </summary>
         public string Note { get; set; }
 
+        /// <summary>
+        /// The plaintext responseText of the puzzle, which is a subset of the responseText when the responseText is of the form:
+        /// {plaintextResponseText}Html.Raw({htmlResponseText})
+        /// </summary>
+        public string GetPlaintextResponseText(int eventId)
+        {
+            string responseText = this.ResponseText;
+            if (responseText != null && responseText.EndsWith(")") && responseText.Contains("Html.Raw("))
+            {
+                responseText = responseText.Replace("{eventId}", $"{eventId}");
+                responseText = responseText.Substring(0, responseText.IndexOf("Html.Raw(")).TrimEnd();
+            }
+            return string.IsNullOrEmpty(responseText) ? this.ResponseText : responseText;
+        }
+        

```

Comment:
This doesn't seem quite right to me. It seems to me that {eventId} should never be left behind, because the only reason it will be there is if it's in a URL, even if Html.Raw isn't used. Note that this doesn't affect my usage.
```
        public string GetPlaintextResponseText(int eventId)
        {
            string responseText = this.ResponseText;
            if (responseText != null) {
                responseText = responseText.Replace("{eventId}", $"{eventId}");
                if (responseText.EndsWith(")") && responseText.Contains("Html.Raw("))
                {
                    responseText = responseText.Substring(0, responseText.IndexOf("Html.Raw(")).TrimEnd();
                }
            }
            return string.IsNullOrEmpty(responseText) ? this.ResponseText : responseText;
        }
```

Reply:
This matches the pattern used elsewhere in the code. If it's ever deemed important to get that scenario working, it'd need a change broader than I'd want to make this close to PH22.

Reply:
I agree with both of you. Let's hold off on that more complex change.

It would probably make sense to move GetPlaintextResponseText into ServerCore.Helpers.RawHtmlHelper so that the places that know about `Html.Raw(...)` are all in the same place. (But not now.)